catalog/spatial/csw/spatial-csw-source-common/src/main/java/org/codice/ddf/spatial/ogc/csw/catalog/common/source/AbstractCswSource.java

```diff
@@ -1718,10 +1686,6 @@ public void setConfigurationPid(String configurationPid) {
         return this.cswSourceConfiguration.getSecurityAttributes();
     }
 
-    public void setMetacardTypes(List<MetacardType> types) {
```

Comment:
@vinamartin  this method only existed to set the private value metacardTypes, which was not used for anything